inspirehep/dojson/utils/__init__.py

```diff
@@ -270,3 +270,11 @@ def split_page_artid(page_artid):
                     page_start = page_artid
 
     return page_start, page_end, artid
+
+
+def force_single_element(obj):
+    """Force an object to a list and return the first element."""
+    lst = force_list(obj)
+    if lst:
+        return lst[0]
+    return None
```

Comment:
Can you randomly return a list though. The behavior of this function is too deterministic.